### PR TITLE
Added Damian

### PR DESCRIPTION
**Pull request by Damovisa**

### CODE REVIEW

File: source/bios/damian_brady.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+authorId: damian_brady
+comments: false
+---
+
+Damian works for [Microsoft](https://microsoft.com/) as a Cloud Developer Advocate in the [League of Extraordinary
+Cloud DevOps Advocates](http://loecda.com) team.
+ 
+An ex-[Microsoft MVP](https://mvp.microsoft.com/en-us/mvp/Damian%20Brady-5000595) and developer at [Octopus Deploy](https://octopus.com), he's also a [Pluralsight Author](https://www.pluralsight.com/authors/damian-brady), and co-author of [Professional Team Foundation Server 2013](http://brdy.in/ProTFS2013) from Wiley. He's based in Toronto Canada, but when back in Australia, he co-organises the [Brisbane .NET User Group](http://www.meetup.com/Brisbane-Net-User-Group/), and the annual [DeveloperDeveloperDeveloper Brisbane](http://www.dddbrisbane.com/) conference.
+ 
+Damian regularly [speaks at conferences, User Groups, and other events](https://damianbrady.com.au/speaking) around the world, but most of the time you'll find him talking to developers and IT Pros to help them get the most out of their DevOps strategies.s
```
Comment (dylan-smith) [inline source/bios/damian_brady.md]: looks like an extra s at the end of this file

Comment (Damovisa) [inline source/bios/damian_brady.md]: Not just that file either - that's what I get for switching to Mac - `Ctrl-S` don't work the same.

File: source/_posts/2017-08-23-Awareness-Acknowledgement-Adjustment.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+3. *Adjustment* of how software is put together - small, self-contained components that can be iterated on more quickly.
+
+**The same pattern can apply for DevOps:**
+
+### DevOps:
+1. *Awareness* that software in prodduction is the aim, and that there are some obvious bottlenecks that slow things down.
+2. *Acknowledgement* that software should be able to help with this - most of what we do is able to be automated!
+3. *Adjustment* by removing the bottlenecks using software and scripts, adding surety through automated testing, and reducing the cycle time from idea to production.
+
+![](/content/images/2017/08/lightbulbs-hanging.jpg)
```
Comment (dylan-smith) [inline source/_posts/2017-08-23-Awareness-Acknowledgement-Adjustment.md]: this image url probably needs an absolute url in there

Comment (Damovisa) [inline source/_posts/2017-08-23-Awareness-Acknowledgement-Adjustment.md]: Ah dammit. Yeah, all of them will.

**Comment by dylan-smith** [comment]
:shipit:

**Comment by dpaquette** [comment]
comb your hair

**Comment by dbelcham** [comment]
holy fucking hippie. @agileramblings is going to feel like someone is trying to steal his persona

**Comment by Damovisa** [comment]
Closing and redoing from a branch in this repo.

**Pull request closed without merging**